python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -370,8 +372,60 @@ def parse_node_data(in_file, feat_ops, node_id_col, label_conf,
             feat_data[key] = val
     return (data[node_id_col], feat_data)
 
+def map_node_ids(src_ids, dst_ids, edge_type, node_id_map, skip_nonexist_edges):
+    """ Map node IDs of source and destination nodes of edges.
+
+    In the ID mapping, we need to handle many errors in the input data. For example,
+    we need to handle the case that endpoint nodes of edges don't exist; we also need
```

Comment:
Can we directly list how many different errors we handle in this function?
`For example` is vague.